src/server/server.rs

```diff
@@ -39,6 +39,8 @@ use super::raft_client::RaftClient;
 use super::metrics::*;
 
 const DEFAULT_COPROCESSOR_BATCH: usize = 50;
+const MAX_GRPC_RECV_MSG_LEN: usize = 8 * 1024 * 1024;
+const MAX_GRPC_SEND_MSG_LEN: usize = 128 * 1024 * 1024;
```

Comment:
_select *_ may need to load a whole region(96M). We can decrease it after introducing streaming for coprocessor requests.